packages/charts/src/charts/donut/donut-chart.component.tsx

```diff
@@ -41,6 +45,8 @@ export type Props = {
   labelsAutocolor?: boolean;
   /** Stack the arcs if percent value is lower than provided treshold */
   stackTreshold?: number;
+  /** Return dataKeys after stacking */
+  onChange?: (res: any) => void;
```

Comment:
onChange is too generic name for that handler in my opinion. Please rename it to something more meaningful like `onStackSeries` or `onDataStack`.